components/PyTorch/pytorch-kfp-components/LICENSE

```diff
@@ -0,0 +1,201 @@
+				Apache License
```

Comment:
Replace with BSD License, use this as reference and just change the "For xxx" line https://github.com/pytorch/tensorpipe/blob/master/LICENSE.txt